app/qml/map/MMMapController.qml

```diff
@@ -92,50 +92,58 @@ Item {
   ]
 
   onStateChanged: {
-    // We call this first because when previous state is 'view' and `centeredToGPS` is true
-    // the map may still not be centered. It's only centered after a certain threshold is exceeded.
-    updatePosition()
-
     switch ( state ) {
+      case "record": {
+        if ( __appSettings.autolockPosition ) { // center to GPS
+          if ( gpsStateGroup.state === "unavailable" ) {
+              __notificationModel.addError( "GPS currently unavailable." )
+              return;
+          }
```

Comment:
We completely disable recording when GPS is unavailable and autolock is on?
In that case maybe we should also handle the case when GPS is lost while recording?

Reply:
Good point, no, we need to continue even without GPS signal